svelte.config.js

```diff
@@ -7,7 +7,10 @@ const config = {
 		// adapter-auto only supports some environments, see https://kit.svelte.dev/docs/adapter-auto for a list.
 		// If your environment is not supported or you settled on a specific environment, switch out the adapter.
 		// See https://kit.svelte.dev/docs/adapters for more information about adapters.
-		adapter: adapter()
+		adapter: adapter(),
```

Comment:
We have another directory called components inside lib directory, added there for easy access with $lib. This is perfectly fine as well, but let's decide on keeping one and remove the other folder. wdyt?